eox_tagging/test/test_models.py

```diff
@@ -118,6 +118,92 @@ def test_empty_setting(self):
                 access=AccessLevel.PRIVATE,
             )
 
+    @override_settings(
```

Comment:
same docstring `Used to test that if a field is `forced` the must be set to the value specified.` please be more specific for each method.

Reply:
sorry. Done